### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by jkotan** [nexpy/nexpy#179]
Trying to plot data from a file [(from here)](https://github.com/jkotan/file-examples/blob/master/pninexus/testp06_cp3.nxs) `testp06_cp3.scan1234.data.p06_xia01` in the tree view I get the following error:

```
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
/usr/lib/python2.7/dist-packages/nexpy/gui/mainwindow.pyc in plot_data(self)
   1382                         pass
   1383                 elif node.is_plottable():
-> 1384                     dialog = PlotDialog(node, parent=self, fmt='o')
   1385                     dialog.show()
   1386                 else:

/usr/lib/python2.7/dist-packages/nexpy/gui/datadialogs.pyc in __init__(self, node, parent, fmt)
    910         self.grid.addWidget(QtWidgets.QLabel('Signal :'), 0, 0)
    911         self.grid.addWidget(self.signal_combo, 0, 1)
--> 912         self.choose_signal()
    913 
    914         self.layout = QtWidgets.QVBoxLayout()

/usr/lib/python2.7/dist-packages/nexpy/gui/datadialogs.pyc in choose_signal(self)
    937             row += 1
    938             self.grid.addWidget(QtWidgets.QLabel("Axis %s: " % axis), row, 0)
--> 939             self.axis_boxes[axis] = self.axis_box(axis)
    940             self.grid.addWidget(self.axis_boxes[axis], row, 1)
    941         while row < self.grid.rowCount() - 1:

/usr/lib/python2.7/dist-packages/nexpy/gui/datadialogs.pyc in axis_box(self, axis)
    954             box.insertSeparator(0)
    955         box.insertItem(0,'NXfield index')
--> 956         if self.default_axes is not None and self.default_axes[axis] in axes:
    957             box.setCurrentIndex(box.findText(self.default_axes[axis]))
    958         else:
```
However, the following command in the Jupyter widget

    testp06_cp3.scan1234.data.p06_xia01.plot()

works well.

**Comment by jkotan** [comment]
Thanks. I agree we should avoid to create inconsistent files.

**Comment by rayosborn** [comment]
This is fixed in v0.10.11 just released.
